.pre-commit-config.yaml

```
@@ -34,10 +34,11 @@ repos:
       - id: yamllint
 
   - repo: https://github.com/tier4/pre-commit-hooks-ros
-    rev: v0.4.0
+    rev: v0.5.1
     hooks:
       - id: prettier-package-xml
```

https://github.com/autowarefoundation/autoware.core/blob/43fd2d3858275a5da710e0947f5188a6c21c8a8c/.pre-commit-config.yaml#L40-L44

```suggestion
    rev: v0.5.1
    hooks:
      - id: prettier-xacro
      - id: prettier-launch-xml
      - id: prettier-package-xml
```

---

@isamu-takagi Could you confirm this, please? :pray: